pkg/apis/core/v1/conversion_test.go

```diff
@@ -638,15 +638,6 @@ func Test_v1_NodeSpec_to_core_NodeSpec(t *testing.T) {
 		{
 			PodCIDR: "ace:cab:deca::/8",
 		},
-		// Both are provided
-		{
-			PodCIDR:  "10.0.1.0/24",
-			PodCIDRs: []string{"10.0.1.0/24", "ace:cab:deca::/8"},
```

Comment:
Maybe this was supposed to be PodCIDRs: []string{"10.0.1.0/24"} ?

Reply:
Thanks , Changed it